src/pages/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Can we try to avoid disabling this ESLint rule?

Reply:
Removed the ESlint rule